microsoft-edge/webview2/release-notes.md

```diff
@@ -104,6 +104,50 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 <!-- end of top matter -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2057-prerelease
+<!-- Prerelease number TBD -->
+
+Release Date: August 23, 2023
+
+[NuGet package for WebView2 SDK 1.0.2057-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2057-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 118.0.2057.0 or higher.<!-- todo: is .0 correct?  usually a nonzero number -->

```

Comment:
@victorhuangwq @peiche-jessica - is ".0" correct?
WebView2 Runtime version 118.0.2057.0
Expect a number eg 
WebView2 Runtime version 118.0.2057.123
like all other sections in this webpage

Reply:
fixed:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/263c53ee-e5ea-4df3-8403-0b7db0ea3f7f)